nav2_simple_commander/nav2_simple_commander/robot_navigator.py

```diff
@@ -268,8 +268,9 @@ def getResult(self):
     def waitUntilNav2Active(self, navigator='bt_navigator', localizer='amcl'):
         """Block until the full navigation system is up and running."""
         self._waitForNodeToActivate(navigator)
-        self._waitForInitialPose()
         self._waitForNodeToActivate(localizer)
+        if localizer == 'amcl':
```

Comment:
Needs to be in the original order 🙂but otherwise LGTM

Reply:
How am I messing up those small PRs :sweat_smile:. Reverted the order but why does it matter?